### ISSUE TITLE
Support local Python libraries

### ISSUE THREAD

**Issue by int00h5525** [dirtbags/moth#79]
Inside of a puzzle or category, I want to be able to import a local Python module. For example, given
```
category/
  1/
    puzzle.py
    somelib.py
```

I want to be able to do the below from `puzzle.py`
```python
import somelib
```

Or, potentially, given
```
category/
  somelib.py
  1/
    puzzle.py
  2/
    puzzle.py
```

I want to be able to do the below in `1/puzzle.py`
```python
import ..somelib
```

**Comment by nealey** [comment]
I've been considering whether we ought to stop building the dev server to support Python modules, and instead make a "helpers to create a mothball-ready output" library for Python, then switched to "generated puzzles/categories are just executables and if you're on Windows we're sorry".

Then te puzzle generator would output some JSON with all the stuff needed for a puzzle. Our existing "here's a related file as a stream" could just be a file descriptor past 3.

**Comment by int00h5525** [comment]
Closed in #89
